### PR TITLE
Dropdown menu

### PR DESCRIPTION
**Pull request by thaitwo**
Can you check over my dropdown menu when you have chance? Thanks! 🍇 ☕️ 🎾 

Sorry I have a lot of other coding changes along with the dropdown. 🍸

### CODE REVIEW

File: public/javascripts/top-nav-dropdown.js [hunk 1]
```diff
@@ -1,15 +1,26 @@
 var $dropdownMenu = $('.dropdown');
 var $dropdownButton = $('#dropdown-button');
+var $faBars = $('.fa-bars');
```
Comment (alexwolfe) [inline public/javascripts/top-nav-dropdown.js]: I'd come up with a better var name, perhaps `$dropdownButton`

File: resources/scss/_background.scss [hunk 2]
```diff
@@ -1,37 +1,38 @@
 /* BACKGROUND IMAGES
 ============================================================== */
 
-.img-featured {
-  background: linear-gradient(rgba(0,0,0,0.4), rgba(0,0,0,0.4)), url("/images/half-dome-night.jpg") no-repeat center center fixed;
+
+
+.bg-featured {
+  background: linear-gradient(rgba(0,0,0, 0.7), rgba(0,0,0, 0.7)), url("/images/half-dome-night.jpg") no-repeat center center fixed;
   background-size: cover;
```
Comment (alexwolfe) [inline resources/scss/_background.scss]: Looks like you forgot to remove cover from this one.

File: resources/scss/_button.scss [hunk 3]
```diff
@@ -52,4 +52,13 @@
       margin: 20px auto;
     }
   }
+
+  @media handheld and (max-width: $breakpoint-small),
+  screen and (max-device-width: $breakpoint-small),
+  screen and (max-width: $breakpoint-small) {
+
+    &.primary {
```
Comment (alexwolfe) [inline resources/scss/_button.scss]: Change this througout to be `&.button-primary` (prefix your classes unless they are .is- .has-, etc.)

File: resources/scss/_hero.scss [hunk 4]
```diff
@@ -46,13 +46,17 @@
   @media handheld and (max-width: $breakpoint-medium),
   screen and (max-device-width: $breakpoint-medium),
   screen and (max-width: $breakpoint-medium) {
-    margin-bottom: 15px;
+    // margin-bottom: 15px;
```
Comment (alexwolfe) [inline resources/scss/_hero.scss]: Delete this comment :)

File: views/includes/_top-nav.ejs [hunk 5]
```diff
@@ -1,12 +1,18 @@
...
 
-  <i id="dropdown-button" class="fa fa-bars fa-2x"></i>
+  <i id="dropdown-button" class="material-icons md-36">menu</i>
 
   <ul class="dropdown">
-	<li><a href="/contact">Contact</a></li>
-    <li><a href="/about">About</a></li>
-    <li><a href="/gallery">Galleries</a></li>
+  	<li>
+	  	<a href="/gallery">Galleries</a>
```
Comment (alexwolfe) [inline views/includes/_top-nav.ejs]: Mind your indents sir :) On all of these links.

**Comment by alexwolfe** [comment]
Address the last comment and it looks good to merge! 👍

**Comment by thaitwo** [comment]
@alexwolfe Alright, should be all good to go. Thanks for merging! ☕️

**Pull request merged**